build.sbt

```diff
@@ -54,3 +54,20 @@ lazy val kafka: Project = (project in file("clients/kafka"))
   .dependsOn(core)
   .enablePlugins(JavaServerAppPackaging)
   .enablePlugins(DockerPlugin)
+
+lazy val postgres: Project = (project in file("clients/postgres"))
+  .settings(commonSettings: _*)
+  .settings(dockerSettings)
+  .settings(Docker / packageName := "mqperf-postgres")
+  .settings(
+    name := "postgres",
+    libraryDependencies ++= Seq(
+      "org.postgresql" % "r2dbc-postgresql" % "0.9.2.RELEASE",
+      "io.r2dbc" % "r2dbc-pool" % "0.9.2.RELEASE",
+      "org.springframework.data" % "spring-data-r2dbc" % "1.5.6",
```

Comment:
I also didn't plan it... :)  but it is the client which handles statements with `in` condition, has simple api and it is quite well documented. First I tried to use r2dbc-postgres directly, but I couldn't bind array values to query params, later I tried jooq client, but i didn't want to play with any code generators, and in case of execution of string sql I still had a problem to bind a list/array value to `in` condition. So, spring was my third choice... and it works.